third_party/2and3/itsdangerous.pyi

```diff
@@ -126,7 +126,7 @@ class JSONWebSignatureSerializer(Serializer):
     def make_signer(self, salt: Optional[Union[Text, bytes]] = ..., algorithm: SigningAlgorithm = ...) -> Signer: ...
     def make_header(self, header_fields: Optional[Mapping[str, Any]]) -> MutableMapping[str, Any]: ...
     def dumps(self, obj: Any, salt: Optional[Union[Text, bytes]] = ...,
-              header_fields: Optional[Mapping[str, Any]] = ...) -> str: ...
+              header_fields: Optional[Mapping[str, Any]] = ...) -> Union[str, bytes]: ...
```

Comment:
From your description it sounds like this should just be `bytes` (which is an alias for `str` in Python 2).